samples/nextjs/src/pages/[[...path]].tsx

```diff
@@ -0,0 +1,93 @@
+import { NextPage, GetStaticPaths, GetStaticProps } from 'next';
+import Error from 'next/error';
+import Layout from '../components/Layout';
+import { SitecoreContext } from '@sitecore-jss/sitecore-jss-nextjs';
+import { SitecorePageProps, extractPath } from '../lib/page-props';
+import componentFactory from '../temp/componentFactory';
+import { configBasedLayoutService as layoutService } from '../lib/layout-service';
+import { configBasedDictionaryService as dictionaryService } from '../lib/dictionary-service';
+
+const SitecorePage: NextPage<SitecorePageProps> = (props) => {
```

Comment:
Since this is TS, function param types should be explicitly declared. Please make sure TS Lint is correctly set up and picking up issues like this.

Reply:
Is this necessary when using generics since the props type is already specified (`<SitecorePageProps>`)? Regardless, there is a separate task to setup proper linting, which includes addressing any errors.